packages/capabilities/readme.md

```diff
@@ -25,3 +33,126 @@ import * as Voucher from '@web3-storage/capabilities/voucher'
+
+You must provide an input object containing a `with` resource URI that matches the constraints in the capability definition, as well as an `nb` object containing any caveats you want to include. If a capability has no caveats defined, or if all the caveats are optional, you may omit the `nb` field from the input.
+
+```ts
+const cap = Store.add.create({
+  with: 'did:key:z6MkwFPNubhwM66HNKeJYtBu1Rv9n1LZdJhbyhLFg97Qr6FG',
+  nb: {
+    link: 'bagbaieraspawtgooy5lptr7loyd3fxjsrgkamre3y6au3ga4df5bkhrxdkmq',
+    size: 20,
+  }
+})
+```
+
+The above would result in an object similar to the following:
+
+```js
+{
+  can: 'store/add',
+  with: 'did:key:z6MkwFPNubhwM66HNKeJYtBu1Rv9n1LZdJhbyhLFg97Qr6FG',
+  nb: {
+    link: 'bagbaieraspawtgooy5lptr7loyd3fxjsrgkamre3y6au3ga4df5bkhrxdkmq',
+    size: 20,
+  }
+}
+```
+
+#### `invoke`
+
+The `invoke` method returns an "invocation" of the capability, which can be executed against a ucanto service.
+
+Like `create`, `invoke` accepts `with` and `nb` fields, and the inputs must match the constraints in the capability definition.
+
+Because invocations are a type of UCAN, you also need to supply some UCAN-related options. At minimum, you need to include the `issuer`, which is a `Signer` capable of signing a UCAN, and `audience`, which identifies the recipient by DID. You can also include any of the optional fields in the interface definition below:
+
+```ts
+interface UCANOptions {
+  audience: Principal
+  lifetimeInSeconds?: number
+  expiration?: UCAN.UTCUnixTimestamp
+  notBefore?: UCAN.UTCUnixTimestamp
+
+  nonce?: UCAN.Nonce
+
+  facts?: Fact[]
+  proofs?: Proof[]
+}
+```
+
+In the example below, we assume that the `loadSigningKey` function returns a valid `Signer` object.
+
+```ts
+import * as DID from '@ipld/dag-ucan/did'
+
+const issuer = loadSigningKey()
+const audience = DID.parse('did:web:web3.storage')
+
+const invocation = Store.add.invoke({
+  issuer,
+  audience,
+  with: 'did:key:z6MkwFPNubhwM66HNKeJYtBu1Rv9n1LZdJhbyhLFg97Qr6FG',
+  nb: {
+    link: 'bagbaieraspawtgooy5lptr7loyd3fxjsrgkamre3y6au3ga4df5bkhrxdkmq',
+    size: 20,
+  }
+})
+```
+
+Note that creating an invocation does not automatically execute it. To send the invocation to a service, you need a ucanto `ConnectionView` configured to access the service, which you can pass into the `execute` method on the invocation object.
+
+```ts
+const result = await invocation.execute(serviceConnection)
+```
+
+#### `delegate`
+
+The `delegate` method allows you to create a ucanto `Delegation`, which allows another principal to invoke the capability.
+
+`delegate` accepts the same input as `invoke`, however the `nb` field is optional. If `nb` is present, the values provided will act as constraints on the invocations that can be made using the delegation. For example, creating a `store/add` delegation with the `size` caveat set to `1048576` would limit invocations made using the delegation to uploads of no more than 1MiB.
```

Comment:
> For example, creating a `store/add` delegation with the `size` caveat set to `1048576` would limit invocations made using the delegation to uploads of no more than 1MiB.

Love calling out the use cases like this. I think over time it may be worth having explicit examples of how to include that `size` caveat using the `delegate` method.

Another use case I anticipate is "How can I delegate the ability to store a specific CID"?